app/views/layouts/_header_not_connected.haml

```diff
@@ -0,0 +1,4 @@
+%ul.nav.navbar-nav.navbar-right
+  - if AppConfig.settings.enable_registrations? && !current_page?(controller: "/registrations", action: :new)
+    %li= link_to t("devise.shared.links.sign_up"), new_user_registration_path, class: 'login'
+  %li= link_to t('devise.shared.links.sign_in'), new_user_session_path, class: 'login'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.